pysoem/pysoem.pyx

```diff
@@ -738,7 +745,10 @@ cdef class CdefSlave:
 
         cdef cpysoem.ec_errort err
         if cpysoem.ecx_poperror(self._ecx_contextt, &err):
-            self._raise_exception(&err)
```

Comment:
Imagine there is an emergency message AND a SDO abort in the error queue, would it maybe be better to call the poperror() function at least twice in case the first "poped" error is the emergency message?

Reply:
I mean, not within `mbx_receive()` but within `sdo_read()` and `sdo_write()`.

Reply:
Done [0254a91](https://github.com/bnjmnp/pysoem/pull/132/commits/0254a919c22c48a688c66e49d6705def654bd081)